app/frontend/src/features/search/SearchBox.tsx

```diff
@@ -1,5 +1,10 @@
 import { makeStyles } from "@material-ui/core";
-import TextField from "components/TextField";
+import FormControl from "@material-ui/core/FormControl";
+import IconButton from "@material-ui/core/IconButton";
+import Input from "@material-ui/core/Input";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import InputLabel from "@material-ui/core/InputLabel";
+import SearchIcon from "@material-ui/icons/Search";
```

Comment:
prefer importing from `@material-ui/core`

Reply:
Did I do it right?